src/iomap.cpp

```diff
@@ -198,9 +198,22 @@ bool IOMap::loadMap(Map* map, const std::string& identifier)
 				setLastErrorString("Invalid map node.");
 				return false;
 			}
-
-			uint16_t base_x = area_coord.x;
-			uint16_t base_y = area_coord.y;
+			
+			uint16_t base_x = 0;
+			uint16_t base_y = 0;
+			uint16_t offset_x = 0;
+			uint16_t offset_y = 0;
+			
+			if (place_x != 0 && place_y != 0) {
+				offset_x = base_x - place_x;
+				offset_y = base_y - place_y;
+				base_x = base_x - offset_x;
+				base_y = base_y - offset_y;
+			}
```

Comment:
Yea you have right. I forgot remove that, becouse first my version been create offset.